test/fuzz/fuzz-unit-file/directives.service

```diff
@@ -855,6 +855,7 @@ PivotRoot=
 Port=
 PowerKeyIgnoreInhibited=
 Private=
+PrivateIPC=
```

Comment:
`IPCNamespacePath=` is missing?

Reply:
`NetworkNamespacePath` is also missing.  Is it intentional?

Reply:
Ouch. Could you also add it?